src/utils/FilterTodos.ts

```diff
@@ -10,28 +10,21 @@ export function getFilteredTodos(
   todos: Todo[],
   { sortField, query }: Filters,
 ): Todo[] {
-  let todosCopy = [...todos];
-
-  if (query) {
-    const normalizedQuery = query.trim().toLowerCase();
-
-    todosCopy = todosCopy.filter(todo => {
+  return [...todos]
```

Comment:
[Not critical]
Copying `todos` array using spread is not needed here:
- `filter` would anyway return a new array
- objects in that array would be the same, since spread operator makes shallow copy